### PR TITLE
Update 040D.lang

### PR DESCRIPTION
**Pull request by dinuto**

### CODE REVIEW

**Comment by micwoj92** [comment]
@HypeGameMode do you approve?

**Comment by HypeGameMode** [comment]
About the language fix, Yes I approve.
But I see you added an hebrew translation where I didn't put, and I will tell you why I didn't put. for here example:
64 Bit CPU=64 Bit CPU
64 Bit OS=64 Bit OS
32 Bit CPU=32 Bit CPU
32 Bit OS=32 Bit OS

Because here if i will put hebrew it will make issues with rtl and will not be easy to read on hebrew, This is why I suggest to leave there on English language
Another example where you put hebrew with english which cause the same issue
DirectX 12 and WDDM 3=DirectX 12 ו-WDDM 3
DirectX 12 and WDDM 2=DirectX 12 ו-WDDM 2
Because English is left to right language and hebrew is right to left language it can be very mess when you combine these 2 languages in the same line, sometimes you can do it good, but sometimes it can be mess 
But except for that I approve

**Comment by dinuto** [comment]
ohh I see...
well it _can_ be done right, but I guess if it will ever be implemented it will be part of a complete RTL support for this app.

I will change back the possibly problematic changes.

File: Langs/040D.lang [hunk 1]
```diff
@@ -39,10 +39,10 @@ Supported=נתמך
 Not Supported=לא נתמך
 Disabled / Not Detected=מושבת / לא זוהה
 Drive(s) Meet Requirements=הכוננ/ים עונה/ים על הדרישות
-TPM Missing / Disabled=TPM לא קיים / מושבת
```
Comment (HypeGameMode) [inline Langs/040D.lang]: TPM Missing / Disabled=TPM לא קיים / מושבת

**Review by HypeGameMode — CHANGES_REQUESTED** [review]

File: Langs/040D.lang (hunk 1 above)
Comment (dinuto) [reply Langs/040D.lang]: This starts with English and continues with Hebrew, so the sentence will shown from left to right.
That's why I reverted this change...

**Comment by micwoj92** [comment]
All seems good now? Will be merged if yes

**Comment by HypeGameMode** [comment]
Yes

**Pull request merged**